docs/examples/ex_ace.rst

```diff
@@ -0,0 +1,94 @@
+.. _exace:
+
+Make ACE SWEPAM behave like OMNI Data
+=====================================
+
+NASA OMNI data (also available from `pysatNASA
+<https://github.com/pysat/pysatNASA>`_) is *the* place to go for solar wind
+(SW) parameters.  However, this data set is not available in real time.
+Fortunately, we can use the `OMNI processing
+<https://omniweb.gsfc.nasa.gov/html/omni_min_data.html>`_ to normalize the ACE
+SWEPAM SW proton density and ion temperature.  This is performed using the
+:py:func:`pysatSpaceWeather.instruments.methods.ace.ace_swepam_hourly_omni_norm`
+function.  The example below uses historic ACE SWEPAM data so that we can
+compare the results to OMNI.
+
+::
+
+
+   import datetime as dt
+   import matplotlib as mpl
+   import matplotlib.pyplot as plt
+
+   import pysat
+   import pysatSpaceWeather as py_sw
+   from pysatNASA.instruments import omni_hro
+
+   ace = pysat.Instrument(inst_module=py_sw.instruments.ace_swepam,
+                          tag='historic', update_files=True)
+   omni = pysat.Instrument(inst_module=omni_hro, tag='1min', update_files=True)
+
+   # Add the OMNI conversion routine to the ACE instrument
+   ace.custom_attach(py_sw.instruments.methods.ace.ace_swepam_hourly_omni_norm)
+
+   # Pick a day with a geomagnetic storm
+   stime = dt.datetime(2014, 3, 26)
+
+   # If needed, download the data
+   if stime not in ace.files.files.index:
+       ace.download(start=stime)
+
+   if stime not in omni.files.files.index:
+       omni.download(start=stime)
```

Comment:
This is probably a bug in pysatNASA, but since omni has monthly files, trying to download a date that does not include the 1st of the month means no files will be downloaded.

Reply:
I just fixed this in https://github.com/pysat/pysatNASA/pull/122